client/app/views/folder.coffee

```diff
@@ -11,6 +11,8 @@ File             = require '../models/file'
 
 BACKSPACE_KEY = 8
 
+Errors = require '../config/errors'
```

Comment:
Good idea to do a more general function to handle upload errors here 👍. But is it not better (if it's possible) to keep this errors object here since it should keep little? For instance:

```coffee
    errors: ->
       folderError: 'modal error firefox dragdrop folder'
       existingFolderError: 'modal error existing folder'
       maxSizeExceeded: 'modal error updoad size exceed'
```